config/application.rb

```diff
@@ -15,6 +15,9 @@ module Hours
   class Application < Rails::Application
     require "hours"
     require "time_series_initializer"
+    require "middleware/rescued_subdomain_elevator"
+
+    config.middleware.use Apartment::Elevators::RescuedSubdomain
```

Comment:
@fatboypunk if we end up using this, here is where we can hook into with single tenant mode, and just `unless Hours.single_tenant?`